src/test/ui/associated-types/bound-lifetime-in-binding-only.elision.stderr

```diff
@@ -2,10 +2,12 @@ error[E0106]: missing lifetime specifier
   --> $DIR/bound-lifetime-in-binding-only.rs:62:23
    |
 LL | fn elision<T: Fn() -> &i32>() {
-   |                       ^ expected lifetime parameter
+   |                       ^
+   |                       |
+   |                       expected lifetime parameter
```

Comment:
maybe remove the "expected lifetime parameter" message completely? It already says "missing lifetime specifier" in the error message